processing/src/main/java/io/druid/query/lookup/LookupReferencesManager.java

```diff
@@ -138,6 +177,10 @@ public boolean remove(String lookupName)
     final LookupExtractorFactory lookupExtractorFactory = lookupMap.remove(lookupName);
     if (lookupExtractorFactory != null) {
       LOGGER.debug("Removing lookup [%s]", lookupName);
```

Comment:
Just noticed this line says "removing" but happens *after* it was removed.  Can we adjust the tense on the verb to line up with the actual timeline?